pkgs/development/libraries/libgphoto2/default.nix

```diff
@@ -14,7 +15,10 @@
 , libxml2
 , gd
 }:
-
+let
+  inherit (nixos {}) config;
+  group = config.programs.gphoto2.group;
```

Comment:
Won't this reference the default value and not the one from the users' nixos config? Maybe it's better to make the option read-only (I think that's possible), so that at least the magic string goes away?